site/accepted_emails.json

```diff
@@ -0,0 +1,4 @@
+{
+    "gmail.com": true,
+    "rpi.edu": true
+}
```

Comment:
Should this instead be in a config file? I'm assuming sysadmins would configure this as needed and any contents in `/site` can be blown away by doing a clean install.

Reply:
These have been moved to Submitty.json config file, same with user_id requirements